ex4.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def border_matrix():
+    """
+    This function creates a 10x10 matrix, in which the elements on the borders will be equal to 1, and inside 0.
+    """
+    vector = np.ones((10, 10))
+    vector[1:-1, -1:1] = 0
```

Comment:
This works great, you can also use numpy.pad for an elegant solution.
https://numpy.org/doc/stable/reference/generated/numpy.pad.html